scanpy/__init__.py

```diff
@@ -30,13 +30,15 @@
 from . import tools as tl
 from . import preprocessing as pp
 from . import plotting as pl
-from . import datasets, logging, queries, settings, external
+from . import datasets, logging, queries, external
 
 from anndata import AnnData
 from anndata import read_h5ad, read_csv, read_excel, read_hdf, read_loom, read_mtx, read_text, read_umi_tools
 from .readwrite import read, read_10x_h5, read_10x_mtx, write
 from .neighbors import Neighbors
-from .settings import set_figure_params
+from ._settings import settings
+
+set_figure_params = settings.set_figure_params
```

Comment:
I'll fix this.

Reply:
Ok, got, you made it a member function...

Reply:
I made it a method for the class in anticipation of making the updated rc_params a dict in the object, then using that dict with `mpl.rc_context`